schema/schema.sql

```diff
@@ -0,0 +1,84 @@
+DROP TABLE IF EXISTS track_tag_assignment;
+DROP TABLE IF EXISTS album_tag_assignment;
+DROP TABLE IF EXISTS tag;
+DROP TABLE IF EXISTS track_purchase;
+DROP TABLE IF EXISTS album_purchase;
+DROP TABLE IF EXISTS country;
+DROP TABLE IF EXISTS track;
+DROP TABLE IF EXISTS album;
+DROP TABLE IF EXISTS artist;
+DROP TABLE IF EXISTS subscriber;
+
+
+CREATE TABLE subscriber (
+    subscriber_id SMALLINT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    email VARCHAR(255) UNIQUE NOT NULL,
+    name VARCHAR(255) NOT NULL
```

Comment:
Maybe VARCHAR(60) for name? It depends on what we allow users to enter as their name on Streamlit too.

Reply:
Alright, I'll leave it as VARCHAR(60) for now.